### DISCUSSION TITLE
Issue with ng build not processing correctly in Apollo template.

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#2136] [PrimeNG Templates]
I am using prime ng Apollo, and the site displays correctly when running ng serve in the development environment. However, when I try to publish it on a local IP using ng build --configuration=production, only a loading screen appears, and the site doesn't display correctly.

Upon checking the contents of /dist, I suspect there might be missing files, but even after multiple attempts with ng build, I get the same result without any specific error messages. I think there might be missing files because in the development environment (/src), there is a /src/app directory, but in /dist/apollo-ng, /dist/apollo-ng/app is not generated.

Do you happen to know any solutions for similar issues?

/* Apollo version */
apollo-ng-17.0.0

/* Angular version */
Angular CLI: 17.0.7
Node: 20.12.1
Package Manager: npm 10.5.0
OS: linux x64

Angular: 17.0.7
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package Version
@angular-devkit/architect 0.1700.7
@angular-devkit/build-angular 17.0.7
@angular-devkit/core 17.0.7
@angular-devkit/schematics 17.0.7
@angular/cdk 17.0.4
@schematics/angular 17.0.7
rxjs 7.8.1
typescript 5.2.2
zone.js 0.14.2

**Comment by ghost** [comment]
It has been solved!

**Comment by AshishFreak** [comment]
hi can you share theme ?
